src/app/pages/services/components/service-nfs/service-nfs.component.ts

```diff
@@ -45,6 +46,8 @@ export class ServiceNfsComponent implements OnInit {
     rpclockd_port: helptext.nfs_srv_rpclockd_port_tooltip,
     udp: helptext.nfs_srv_udp_tooltip,
     userd_manage_gids: helptext.nfs_srv_16_tooltip,
+    servers: helptext.nfs_srv_servers_tooltip,
+    // servers: helptext.nfs_srv_servers_placeholder,
```

Comment:
Good idea to review your code before creating a PR 😛

Reply:
👍